### PR TITLE
Add `--faces-list` flag

### PR DESCRIPTION
**Pull request by ghazalrafiei**
#### Reference Issues/PRs
#55 

#### What does this implement/fix? Explain your changes.
Added the `--face-list` flag, showing all available faces for showing the time.

### CODE REVIEW

**Comment by sadrasabouri** [comment]
Thanks for your efforts.
Your `dev` was not originated from `sepandhaghighi:dev`, please solve the conflicts by either merging `sepandhaghighi:dev` or redo the process this time extracting branch from `sepandhaghighi:dev` instead of `sepandhaghighi:master`.

File: mytimer/functions.py [hunk 1]
```diff
@@ -313,6 +313,14 @@ def countdown_timer(
         time.sleep(max(0, 1 - (end - start)))
 
 
+def show_faces_list():
+    """Print all faces samples."""
```
Comment (sadrasabouri) [inline mytimer/functions.py:317]:
```
    """
    Show faces list.
    
    :return: None
    """
```

File: mytimer/functions.py [hunk 2]
```diff
@@ -313,6 +313,14 @@ def countdown_timer(
         time.sleep(max(0, 1 - (end - start)))
 
 
+def show_faces_list():
```
Comment (sadrasabouri) [inline mytimer/functions.py:316]: You can move this function next to the `show_programs_list` for more readability and cohesion.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -45,6 +45,9 @@ jobs:
     - name: Run6
       run: |
         mytimer --minute=0 --second=5 --countdown --alarm --face=3 --tone=2 --alarm-repeat=5
+    - name: Face Lists
```
Comment (sadrasabouri) [inline .github/workflows/test.yml:48]: Move it next to the `Run7` and rename it to `Run8`

File: mytimer/functions.py [hunk 4]
```diff
@@ -313,6 +313,14 @@ def countdown_timer(
...
+    """Print all faces samples."""
+    for i, face in FACES_MAP.items():
+        print('Face %d' %(i))
+        tprint('\t\t\t\t  12 : 34 : 54 ', font=face)
+        print('='*130)
```
Comment (sadrasabouri) [inline mytimer/functions.py:321]: You need to define and use string templates in `params.py`. Just like `show_programs_list`.
Not sure about that.
I suggest:
```python
        tprint('12 : 34 : 54', font=face)
        print('=' * 20)
```
what do you think @sepandhaghighi ?

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Thanks for your efforts and fast response Ghazal.
Take a look at my comments.

File: mytimer/__main__.py [hunk 5]
```diff
@@ -39,6 +39,7 @@ def main():
         const=1)
     parser.add_argument('--countup', help='countup timer', nargs="?", const=1)
     parser.add_argument('--alarm', help='alarm', nargs="?", const=1)
+    parser.add_argument('--faces-list', help='list of available faces', nargs="?", const=1)
```
Comment (sepandhaghighi) [inline mytimer/__main__.py:42]: help='faces list'

File: mytimer/functions.py [hunk 6]
```diff
@@ -313,6 +313,14 @@ def countdown_timer(
         time.sleep(max(0, 1 - (end - start)))
 
 
+def show_faces_list():
+    """Print all faces samples."""
+    for i, face in FACES_MAP.items():
+        print('Face %d' %(i))
```
Comment (sepandhaghighi) [inline mytimer/functions.py:319]: I suggest to user `.format` instead of `%d`

File: mytimer/functions.py (hunk 4 above)
Comment (sepandhaghighi) [reply mytimer/functions.py:321]: My suggestion :

```python
tprint('12 : 34 : 56', font=face)
print('=' * 80)
 ```
 
 1. Remove that `\t\t\t\t`
 2. Move `'12 : 34 : 56'` to `params.py` as `FACES_LIST_EXAMPLE_MESSAGE = "12 : 34 : 56"`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@ghazalrafiei 👋
Thanks for your efforts 💯 
Please take a look at our comments 🔥

**Review by sadrasabouri — APPROVED** [review]
LGTM :rocket:

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**